code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -251,17 +251,13 @@
 	L.SetStuttering(4 SECONDS)
 
 	ADD_TRAIT(L, TRAIT_WAS_BATONNED, user_UID) // so one person cannot hit the same person with two separate batons
-	L.apply_status_effect(STATUS_EFFECT_DELAYED, 2 SECONDS, CALLBACK(L, TYPE_PROC_REF(/mob/living, KnockDown), knockdown_duration), COMSIG_LIVING_CLEAR_STUNS)
 	addtimer(CALLBACK(src, PROC_REF(baton_delay), L, user_UID), 2 SECONDS)
 
 	SEND_SIGNAL(L, COMSIG_LIVING_MINOR_SHOCK, 33)
 
 	L.lastattacker = user.real_name
 	L.lastattackerckey = user.ckey
-	L.visible_message("<span class='danger'>[user] has stunned [L] with [src]!</span>",
-		"<span class='userdanger'>[L == user ? "You stun yourself" : "[user] has stunned you"] with [src]!</span>")
 	add_attack_logs(user, L, "stunned")
-	playsound(src, 'sound/weapons/egloves.ogg', 50, TRUE, -1)
```

Comment:
Does this sound play twice or something? I feel like this sound is a good feedback that they were hit

Reply:
You'll hear the baton actually hitting them still, this just removes the feedback that makes it pretty much the same as a melee baton strike

Reply:
So unless they noticed the baton flying towards them, why won't have any other feedback? I think some kind of text/sound would be good

Reply:
They hear the baton hitting them, just not the additional sound effect

Reply:
I know, but thats like the same sound as anything hitting them, its not really clear feedback that it did anything different than a normal object hitting them